### ISSUE TITLE
v5.0.0 failing to import Prettier plugins (`ERR_MODULE_NOT_FOUND`)

### ISSUE THREAD

**Issue by stayradiated** [hyperfocalHQ/pollen#96]
Hi there, 

Thank you for Pollen, it's such a great project and I really enjoy using it :raised_hands: 

**Error Description**

I recently tried upgrading to v5 - but when I run `pnpm exec pollen` I get this error:

```
node:internal/modules/esm/resolve:260
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/my-project/node_modules/.pnpm/pollen-css@5.0.0/node_modules/pollen-css/node_modules/.pnpm/prettier@3.3.2/node_modules/prettier/plugins/postcss.mjs' imported from /my-project/node_modules/.pnpm/pollen-css@5.0.0/node_modules/pollen-css/dist/index.js
    at finalizeResolution (node:internal/modules/esm/resolve:260:11)
    at moduleResolve (node:internal/modules/esm/resolve:920:10)
    at defaultResolve (node:internal/modules/esm/resolve:1119:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:541:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:510:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleLoader.import (node:internal/modules/esm/loader:473:34)
    at defaultImportModuleDynamicallyForModule (node:internal/modules/esm/utils:214:31)
    at importModuleDynamicallyCallback (node:internal/modules/esm/utils:253:12)
    at importPlugin (file:///my-project/node_modules/.pnpm/pollen-css@5.0.0/node_modules/pollen-css/dist/index.js:28770:25) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///my-project/node_modules/.pnpm/pollen-css@5.0.0/node_modules/pollen-css/node_modules/.pnpm/prettier@3.3.2/node_modules/prettier/plugins/postcss.mjs'
}
```

**Potential Cause**

I am looking at the `dist/index.js` file inside the `pollen-css` NPM package, at the line that throws the above error:

```javascript
    importPlugin: () => import('../node_modules/.pnpm/prettier@3.3.2/node_modules/prettier/plugins/postcss.mjs'),
```

It feels like the build process might be doing something wrong here, I wouldn't have expected it to hard code an absolute path to the `node_modules/.pnpm` directory like this. 

**Versions**

Node: v22.3.0
pnpm: v9.4.0
prettier: v3.3.2
pollen: v5.0.0

**Comment by madeleineostoja** [comment]
Ah I think I know what this might be. The Rollup build for Pollen maintains `import()` statements rather than bundling them in order to lazyload the (heavy) css vars polyfill dynamically when using the `shimmie` utility only if its required in IE. My guess is that other deps (prettier in this case) is using them and it's leading to werido behaviour. 

If that's the case then the easy fix is to just remove that behaviour, it's an edge case that I wouldn't mind hard-deprecating at some point anyway, especially since IE11 is now at end of life.

**Comment by madeleineostoja** [comment]
Fixed in v5.0.1

**Comment by stayradiated** [comment]
Brilliant, v5 is working for me now - thank you! :raised_hands:
